db/migrate/20220809145929_add_post_id_to_comments.rb

```diff
@@ -0,0 +1,6 @@
+class AddPostIdToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_column :comments, :PostId, :integer
+    add_index :comments, :PostId
```

Comment:
- [ ]  Also same here trying to add a column won't create the relationship between comments and the post 
to solve that you should use the following command : 
```RUBY
t.references :post, foreign_key: true
```

- [ ]   please it would be better to follow the convention of creating columns in the database they should be in snake_case like that: comment_counter ...etc please try to do that